nf_core/pipeline-template/workflows/pipeline.nf

```diff
@@ -120,6 +120,7 @@ workflow.onComplete {
     if (params.email || params.email_on_fail) {
         NfcoreTemplate.email(workflow, params, summary_params, projectDir, log, multiqc_report)
     }
+    NfcoreTemplate.dump_parameters(params)
```

Comment:
Do you think it would be good to mention that this can be added to the MultiQC report? Maybe in a TODO comment

Reply:
In the current implementation, it won't be possible to add it to the multiqc report since is invoked by the `workflow.onComplete` event. Also, the idea in this case is more to generate a `json` file that can be used to run the pipeline using exactly the same parameters. Although, it might be also interesting what you suggest, in this case probably we will need to add more logic to clean a little the parameters that are displayed.

Reply:
I see, thanks for the explanation! I was confused by the issue description